.github/workflows/changelog.yml

```diff
@@ -0,0 +1,93 @@
+# This action requires that any PR targeting the main branch should add a
```

Comment:
nit: `This action requires that any PR targeting the aws-cwa-dev branch`

Reply:
changed.